javascript/modules/client/test/integration/client-addressList.test.ts

```diff
@@ -422,6 +425,116 @@ describe("Client", () => {
       expect(installPluginItemItem.data).toBeInstanceOf(Uint8Array);
       expect(installPluginItemItem.to).toBe(pluginAddress);
     });
+
+    it("Should encode a add members action with an invalid plugin address and fail", async () => {
+      const context = new ContextPlugin(contextParamsLocalChain);
+      const client = new ClientAddressList(context);
+
+      const members: string[] = [
+        "0x1357924680135792468013579246801357924680",
+        "0x2468013579246801357924680135792468013579",
+        "0x0987654321098765432109876543210987654321",
+      ];
+
+      const pluginAddress = "0xinvalid_address";
+      expect(() =>
+        client.encoding.addMembersAction(
+          pluginAddress,
+          members,
+        )
+      ).toThrow(new InvalidAddressError());
+    });
+    it("Should encode a add members action with an invalid member address and fail", async () => {
+      const context = new ContextPlugin(contextParamsLocalChain);
+      const client = new ClientAddressList(context);
+
+      const members: string[] = [
+        "0xinvalid_address",
+        "0x2468013579246801357924680135792468013579",
+        "0x0987654321098765432109876543210987654321",
+      ];
+
+      const pluginAddress = "0x1234567890123456789012345678901234567890";
+      expect(() =>
+        client.encoding.addMembersAction(
+          pluginAddress,
+          members,
+        )
+      ).toThrow(new InvalidAddressError());
+    });
+    it("Should encode a add members action", async () => {
+      const context = new ContextPlugin(contextParamsLocalChain);
+      const client = new ClientAddressList(context);
+
+      const members: string[] = [
+        "0x1357924680135792468013579246801357924680",
+        "0x2468013579246801357924680135792468013579",
+        "0x0987654321098765432109876543210987654321",
+      ];
+
+      const pluginAddress = "0x1234567890123456789012345678901234567890";
+      const action = client.encoding.addMembersAction(pluginAddress, members);
+
+      expect(typeof action).toBe("object");
+      expect(action.data).toBeInstanceOf(Uint8Array);
```

Comment:
Maybe it's excessive but I would suggest to convert `action.data` into a hex string, and compare it to the hardcoded expected value